lib/adb/adbcontroller.py

```diff
@@ -13,4 +13,24 @@ def run_command(id, *command):
         return output
 
 
+    def make_phone_call_adb(self,device_id,number):
+        """
+        It receives a telephone number and makes the call with adb
+        Pre-condition: the number has to be valid
+        param str numner: telephone number
+        format number: "1234567890"
+        returns: 
```

Comment:
En la línea 30 regresas false, pero si la ejecución es correcta, no debería de regresar algo también?

Reply:
```suggestion
        Phone_number =''.join(i for i in number if i.isdigit() or i=="+")
        if len(Phone_number) == 13:
            return True
        else:
            raise ValueError("Incorrect Format number")
```
Lo hice de esta manera para cubrir el marcar cualquier numero mexicano o numero de paises que tengan un largo de 12 digitos, ya que estuve investigando y hay muchos formatos y ladas diferentes para marcar en todos los paises. Este escenario cubre el marcar de cualquier parte del mundo, a un numero mexicano que es usando el "+52". si es correcto el numero regresa un true y sino, una excepcion de ValueError es lanzada. Es correcto?